muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/coursepicker/CoursePickerWorkspace.java

```diff
@@ -154,6 +165,7 @@ public void setOrganization(OrganizationRESTModel organization) {
   private Boolean published;
   private Boolean isCourseMember;
   private String educationTypeName;
+  private Mandatority educationSubtypeName;
```

Comment:
Would be better if this property was named `mandatority`. It is rather misleading if `educationTypeName` is a `String` whereas `educationSubtypeName` is `Mandatority`